scripts/src/java/org/oppia/android/scripts/docs/BUILD.bazel

```
@@ -1,18 +1,17 @@
 """
-Libraries corresponding to kdoc presence check, that ensures all public members (except overrides)
-must have a corresponding KDoc.
+Libraries corresponding docs validity check, that ensures the documentations are valid.
```

```suggestion
Libraries corresponding to documentation-related scripts.
```

Can be simpler. Ditto for the tests version.

---

Done.